docs/integrations.md

```diff
@@ -59,9 +58,31 @@ jobs:
       #  if: always()
 ```
 
-</p>
-</details>
-<details><summary>GitLab CI</summary>
+<a name="inline-pr-comments-beta"></a>
+<br />
+
+#### Automatic PR Comments (beta)
+
+!!! info
+    This feature is currently only available for GitHub.
+
+To get inline PR comments on your pull requests, set the `GITHUB_TOKEN` environment variable in your workflow file to `secrets.GITHUB_TOKEN`, which is the GitHub app installation access token and takes the form of this snippet:
+
+```
+uses: returntocorp/semgrep-action@v1
+        env: # Optional environment variable for inline PR comments (beta)
```

Comment:
```suggestion
        env: # Optional environment variable for automatic PR comments (beta)
```

Likely also needs to be changed in the sample GitHub Action workflow file above.